app/templates/suppliers/details.html

```diff
@@ -188,11 +188,12 @@
       <div class='column-two-thirds'>
         <form method="POST" action="{{ url_for('.confirm_supplier_details') }}">
           <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
-          <p>Once you confirm your information you'll need to contact support to change your:</p>
+          <p>Once you confirm your information you'll need to contact support to correct a mistake in your:</p>
```

Comment:
Should this be `contact CCS`?

Reply:
No, it'll still need to come through support. And they'll get the email address when they actually click the link on the thing when they want to update it.

Reply:
Right, but it's a CCS email address.

Eh, ok 🤷‍♂️